liquibase/dbchangelog.xml

```diff
@@ -0,0 +1,113 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd">
+
+    <changeSet id="1" author="jossuecito">
+        <!-- Dependencies table -->
+        <createTable tableName="fileurls">
+            <column name="id" type="int">
+                <constraints primaryKey="true" primaryKeyName="pk_fileurls"/>
+            </column>
+            <column name="url" type="varchar(2000)">
+                <constraints nullable="false"/>
+            </column>
+            <column name="rawdata" type="clob">-->
```

Comment:
Do we know on average how large the rawdata payloads are? We could potentially use MEDIUMTEXT or LONGTEXT here instead of CLOB. WDYT?

Reply:
These are Liquibase data types, since internally a conversion is done we cannot have specific implementations data types.
This is the data types mapping table
https://dba-presents.com/index.php/liquibase/216-liquibase-3-6-x-data-types-mapping-table